plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/config/Config.java

```diff
@@ -10,12 +10,16 @@ public class Config {
     private final List<String> plugins;
     private final List<String> recipes;
     private final Path cachePath;
+    private final String mavenHome;
+    private final String mavenPluginVersion;
```

Comment:
mavenRewritePluginVersion is more explicit and clear that is the version of the rewrite maven plugin